docs/pbc/all/multi-factor-auth/202505.0/multi-factor-auth.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Multi-Factor Authentication
+description: Learn about Multi Factor Authentication (MFA) that you can use within your Spryker project.
+template: concept-topic-template
+last_updated: Mar 05, 2026
+---
+
+{% info_block warningBox "⚠️ WARNING: BETA FEATURE" %}
+
+**THE MULTI-FACTOR AUTH MODULE IS CURRENTLY IN BETA.**
+
+This feature is not yet considered stable and **may be significantly changed in future releases**.
+
+By using this feature, you accept full responsibility for any potential issues, including breaking changes, limited support, and incomplete functionality.
+
+**Use in production environments is not recommended** unless you are prepared to adapt to future updates.
+
+{% endinfo_block %}
+
+Multi-Factor Authentication adds an extra layer of security to your account.
+It allows customers to enhance their account security by requiring additional verification steps beyond the standard username and password.
+
+## Benefits of MFA
+
+- **Enhanced security**: MFA adds an extra layer of security to your account, making it harder for attackers to gain access.
+- **Enhanced privacy**: MFA helps protect your privacy by requiring additional verification steps beyond the standard username and password.
+- **Reduced risk**: MFA reduces the risk of unauthorized access, as it requires multiple factors to be verified before allowing access.
+- **Compliance**: MFA helps you comply with security regulations and standards.
+
+## Flow Overview
```

Comment:
What flow is described here? Compared to the flow in the next section that is

Reply:
Here is the general flow applied for all forms configured as secured. 
Below is about the only Login form where the flow is a bit different. First, you have to pass credentials validation (e-mail + password), and only after that, if you succeed, the MFA validation is triggered. For all other forms, the first step is skipped as they're placed in the Customer Profile.